tests/package.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  newton_raphson:
```

Comment:
Kebab?

Reply:
Leaving as is, as this is done by 'toit pkg'.